crates/pop-common/src/sourcing/errors.rs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
A whole module for a single enum seems unnecesssary, especially when it is named errors but there is/will probably only ever be a single error type. This pattern has developed from previous implementations and not sure it should be followed. May be subjective however.